holidays/countries/russia.py

```
@@ -118,6 +124,35 @@ class RussiaStaticHolidays:
+    christmas_day = tr("Рождество Христово")
+
+    defender_of_fatherland_day = tr("День защитника Отечества")
+
+    international_womens_day = tr("Международный женский день")
+
+    labor_day = tr("Праздник Весны и Труда")
+
+    victory_day = tr("День Победы")
+
+    declaration_of_sovereignty_day = tr(
+        "День принятия Декларации о государственном суверенитете Российской Федерации"
```

Ruff (0.15.1)

[warning] String contains ambiguous `о` (CYRILLIC SMALL LETTER O). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)